.gitignore

```diff
@@ -1 +1,2 @@
 /target
+.vscode/settings.json
```

Comment:
Is there a newline at the end of this file? Github's review UI is showing me a little red circle with a line in it, which I _think_ may mean there's not?